release/tune_tests/cloud_tests/tpl_gce_4x8.yaml

```diff
@@ -7,11 +7,11 @@ max_workers: 3
 
 head_node_type:
     name: head_node
-    instance_type: n2-standard-8
+    instance_type: n2-standard-2
```

Comment:
is this instance type change related to this PR's topic?

Reply:
I used 8 previously which mimics gce_k8 test setting. aws uses 2 though so I change this as well for consistency with aws test.